gobblin-cluster/src/main/java/org/apache/gobblin/cluster/SingleTaskLauncher.java

```diff
@@ -56,6 +56,9 @@ Process launch(final String jobId, final Path workUnitFilePath)
 
     // The -cp parameter list can be very long.
     logger.debug("cmd: " + command);
+    final String completeCmdLine = String.join(" ", command);
+    logger.debug("cmd line: \n" + completeCmdLine);
```

Comment:
Same here. Use of {} is more efficient especially since debug usually won't be emitted.

Reply:
Done. Thanks